### PR TITLE
Moved controller constants into Controllers class.  Replaced Meters a…

### PR DESCRIPTION
**Pull request by rma-rripken**
…nd Timers with markAndTime method call.

### CODE REVIEW

File: cwms_radar_api/src/test/java/cwms/radar/api/SpecifiedLevelIdIT.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public class SpecifiedLevelIdIT extends DataApiTestIT {
     public static final String OFFICE = "SPK";
 
     @Test
-    public void test_create_read_delete() throws JsonProcessingException {
+    void test_create_read_delete() throws JsonProcessingException {
```
Comment (MikeNeilson) [inline cwms_radar_api/src/test/java/cwms/radar/api/SpecifiedLevelIdIT.java:51]: Test functions should be public. Otherwise they might not get picked up.

**Review by MikeNeilson — CHANGES_REQUESTED** [review]
Other than one issue on a test this looks great.

File: cwms_radar_api/src/test/java/cwms/radar/api/SpecifiedLevelIdIT.java (hunk 1 above)
Comment (adamkorynta) [reply cwms_radar_api/src/test/java/cwms/radar/api/SpecifiedLevelIdIT.java:51]: JUnit5 no longer requires public tests, they just can't be private. JUnit4 also removed the requirement of starting the method name with "test" but I can't kick the habit.

Comment (MikeNeilson) [reply cwms_radar_api/src/test/java/cwms/radar/api/SpecifiedLevelIdIT.java:51]: fair. and yeah i generally do the test_ as a matter of abundantly clear intent that "this function is testing something specific and not support."

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**